### DISCUSSION TITLE
Focus issue after date range picker onClean

### DISCUSSION THREAD

**Discussion by kimberlychan1** [rsuite/rsuite#2489] [Q&A]
I have encountered an issue that after I cleared the value of the date picker and clicked submit, the previous value will still be there. However, if I clear the value and click somewhere else before clicking submit, the value will be updated successfully. May I know besides creating a dummy element to focus when onClean, is there a better solution for it? Thank you

**Comment by SevenOutman** [comment]
@kimberlychan1 Could you provide a CodeSandbox demo? You may use this template https://codesandbox.io/s/rsuite-template-5vq6zo2z5l
